rts/Lua/LuaOpenGL.cpp

```diff
@@ -8,10 +8,14 @@
 //    requires the ARB_imaging extension)
 // - use materials instead of raw calls (again, handle dlists)
 
-#include "Rendering/GL/myGL.h"
-#include <string>
+//#include "Rendering/GL/myGL.h"
+
 #include <vector>
 #include <algorithm>
+#include <optional>
+
+#include "lib/fmt/format.h"
+#include "lib/sol2/sol.hpp"
```

Comment:
why was sol.hpp included in this file?
also, why was optional included? the only mention of optional I've seen in this PR (outside of this file) are sol::optional and surely that doesn't require the include of the std's header?